src/js/editor/editor.js

```diff
@@ -754,8 +754,7 @@ class Editor {
     let pastedPost = parsePostFromPaste(event, this.builder, this._parserPlugins);
 
     this.run(postEditor => {
-      let nextPosition = postEditor.insertPost(position, pastedPost);
-      postEditor.setRange(new Range(nextPosition));
```

Comment:
Cool. I'm not sure if we had the discussion or not. I know having mutation methods with the side effect of changing cursor position is something we've avoided until now. However the arguments against it are fewer now that we have `setRange`, a safe way to set the position and read it back without the DOM.